Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,195 @@
+            nuevaVenta.FacturaId = int.Parse(Console.ReadLine());
+            Console.WriteLine("Ingrese codigo del producto :");
+            nuevaVenta.ProductoId = int.Parse(Console.ReadLine());
+            Console.WriteLine("Ingrese codigo del vendedor :");
+            nuevaVenta.VendedorId = int.Parse(Console.ReadLine());
+            Console.WriteLine("Ingrese codigo del cliente :");
+            nuevaVenta.ClienteId = int.Parse(Console.ReadLine());
+            Console.WriteLine("Ingrese descripcion de la venta :");
+            nuevaVenta.descripcion = Console.ReadLine();
+            ArraynuevaVenta.Add(nuevaVenta);
+            Console.WriteLine("Cliente registrado exitosamente");
+        
+        
+        }
+        public static void registroVendedores(){
+
+            List<Vendedor> ArrayvendedorNuevo = new List<Vendedor>();
+            Vendedor vendedorNuevo = new Vendedor();
+            Console.WriteLine("--------------------");
+            Console.WriteLine("Registro de Vendedores");
+            Console.WriteLine("--------------------");
+            Console.WriteLine("Ingrese codigo :");
+            vendedorNuevo.VendedorId = int.Parse(Console.ReadLine());
+            Console.WriteLine("Ingrese nombres :");
+            vendedorNuevo.Nombre = Console.ReadLine();
+            Console.WriteLine("Ingrese apellidos :");
+            vendedorNuevo.Apellido = Console.ReadLine();
+            Console.WriteLine("Ingrese correo :");
+            vendedorNuevo.Cargo = Console.ReadLine();   
+            ArrayvendedorNuevo.Add(vendedorNuevo);         
+            Console.WriteLine("Vendedor registrado exitosamente");
+        }
+
+
+        public static void registroProductos(){
+            List<Producto> ArrayproductoNuevo = new List<Producto>();
+            Producto productoNuevo = new Producto();
+            Console.WriteLine("--------------------");
+            Console.WriteLine("Registro de Productos");
+            Console.WriteLine("--------------------");
+            Console.WriteLine("Ingrese codigo :");
+            productoNuevo.ProductoId = int.Parse(Console.ReadLine());
+            Console.WriteLine("Ingrese nombre :");
+            productoNuevo.NombreProducto = Console.ReadLine();
+            Console.WriteLine("Ingrese stock :");
+            productoNuevo.stock = int.Parse(Console.ReadLine());
+            Console.WriteLine("Ingrese precio :");
+            productoNuevo.Precio = int.Parse(Console.ReadLine());
+            ArrayproductoNuevo.Add(productoNuevo);
+            Console.WriteLine("Producto registrado exitosamente");
+        }
+    }
+
+     public class Producto
+    {
+        public int ProductoId { get; set; }
+        public string NombreProducto { get; set; }
+        public double Precio { get; set; }
+        public string DescripcionProducto { get; set; }
+        public int stock { get; set; }
+        const int minimo_stock = 1000;

```

Comment:
At coding standards says "Use all caps for constants, separating words with an underscore."
Also declaration of constants must to be at the beginning.